app/src/main/AndroidManifest.xml

```diff
@@ -192,7 +192,7 @@
 
```

Comment:
This file must be replicated into main and full. On main, it should be the common parts and on full the current AndroidManifest. At compile-time Gradle merges both files, adding missing parts. It makes a diff and creates a new one.

On main, the manifest must have any Google related section stripped. On full it has to have those section. On foss there is no need to have the manifest there, as it is in fact the /main one

Reply:
Remove the billing permission from /main